src/main/java/com/nokia/uwr/service/parser/BTSParserImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.nokia.uwr.service.parser;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nokia.uwr.scenario.bts.BTSDescription;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Main implementation
+ *
+ * @author Piotr Stoklosa
+ */
+@Service
+public class BTSParserImpl implements BTSParser {
+
+    private static final Logger LOGGER = LogManager.getLogger(BTSParserImpl.class);
+
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
We should make it possible to be injected by constructor. Moreover, ObjectMapper could be a bean.